kuksa_client/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright Robert Bosch GmbH, 2020. Part of the Eclipse Kuksa Project.
+#
+# All rights reserved. This configuration file is provided to you under the
+# terms and conditions of the Eclipse Distribution License v1.0 which
+# accompanies this distribution, and is available at
+# http://www.eclipse.org/org/documents/edl-v10.php
+#
+
+# Note: This dockerfile needs to be executed one level above in the root folder
+
+FROM python:3.8-alpine as build
+# TODO build and upload
+RUN apk update && apk add git bash
+ADD . /kuksa.val
+WORKDIR /kuksa.val
+RUN pip install --upgrade pip build
+RUN python -m build
+
+FROM python:3.8-alpine
+
+COPY --from=build /kuksa.val/dist /kuksa_dist
+RUN pip install kuksa_dist/*.whl 
```

Comment:
Optimization: Would be more effcient to install in "build" dev and only copy files. Here the wheel always is part of the docker...

Reply:
I do not know to install in "build" dev. However, the kuksa_dist will be removed in the next line

Reply:
Docker is "like git", due to the layers. Not matter you delete it or not, it always takes up space in the final image.
See testclient docker for an example https://github.com/eclipse/kuksa.val/blob/master/clients/vss-testclient/Dockerfile all python packages are installed locally, and then just the folder is copied.

Reply:
ok, I got it running now